jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -186,24 +187,37 @@ def search(self):
 
         # markers are added to the viewer based on the table
         viewer.marker = {'color': 'red', 'alpha': 0.8, 'markersize': 5, 'fill': False}
-        viewer.add_markers(table=catalog_results, use_skycoord=True, marker_name='catalog_results')
+        viewer.add_markers(table=catalog_results, use_skycoord=True, marker_name=self._marker_name)
 
         return skycoord_table
 
     def vue_do_search(self, *args, **kwargs):
         # calls self.search() which handles all of the searching logic
         self.search()
 
-    def clear(self):
-        if self.results_available:
+    def clear(self, hide_only=True):
```

Comment:
this isn't public API yet, so we can defer this decision if you prefer, but I wonder if separate `hide` and `clear/remove` methods would be more intuitive to the user (and then we could have the UI button always call `hide`)?